build/makelib/helm.mk

```diff
@@ -20,7 +20,7 @@ HELM_CHARTS_DIR ?= $(ROOT_DIR)/cluster/charts
 HELM_OUTPUT_DIR ?= $(OUTPUT_DIR)/charts
 
 HELM_HOME := $(abspath $(CACHE_DIR)/helm)
-HELM_VERSION := v2.16.7
+HELM_VERSION := v3.2.1
```

Comment:
Why not latest v3.4.0?

Reply:
Will update to the latest, so far I just opened it in the same state as the original PR (rebased) to see where the current state of the CI is. Now to investigate the helm failures...